utils/carousel.ts

```diff
@@ -45,6 +51,36 @@ export const formatNFT = async (
   })
 }
 
+export const setNftMetaFromCache = async (nfts): Promise<CarouselNFT[]> => {
+  return await Promise.all(
```

Comment:
do we still need to keep Promise.all?

Reply:
Yes, because we need array of NFT in `useCarousel`. It will be array of promises without `Promise.all`